python/CMakeLists.txt

```diff
@@ -146,26 +218,39 @@ endforeach()
 #
 # Install revng.daemon
 #
-set(REVNG_DAEMON_MODULE_FILES
-    revng/daemon/__init__.py revng/daemon/multiqueue.py
-    revng/daemon/event_manager.py revng/daemon/schema.graphql
-    revng/daemon/graphql.py revng/daemon/util.py)
-python_module(TARGET_NAME revng-python-daemon MODULE_FILES
+file(
+  GLOB REVNG_DAEMON_MODULE_FILES
+  RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
+  revng/internal/daemon/*.py revng/internal/daemon/*.graphql)
```

Comment:
Never use `GLOB`.
Developers must know that they have to list files if they want them to be picked up.
If you use `GLOB` you don't need to *but* you need to re-run configure which is very confusing.
Also, `GLOB` picks up scratch files you might have around.

No `GLOB`.

Reply:
Dropped `GLOB`